__mocks__/renderWithRedux.ts

```diff
@@ -0,0 +1,10 @@
+import { RenderOptions, render } from "@testing-library/react";
+import ReduxProvider from "./provider/ReduxProvider";
+import { ReactElement } from "react";
+
+const customRender = (ui: ReactElement , options?: Omit<RenderOptions, "wrapper">) => {
```

Comment:
I think using **ui**, limit the use of the provider to being used with only ui elements.